src/main/java/com/synopsys/integration/alert/provider/blackduck/tasks/ProjectSyncTask.java

```diff
@@ -108,18 +108,30 @@ public void run() {
         logger.info("### Finished {}...", getTaskName());
     }
 
-    public Map<BlackDuckProject, ProjectView> getCurrentData(final List<ProjectView> projectViews) {
-        final Map<BlackDuckProject, ProjectView> projectMap = projectViews
-                                                                  .stream()
-                                                                  .collect(
-                                                                      Collectors.toMap(projectView -> new BlackDuckProject(projectView.name, StringUtils.trimToEmpty(projectView.description), projectView._meta.href), Function.identity()));
+    public Map<BlackDuckProject, ProjectView> getCurrentData(final List<ProjectView> projectViews, HubService hubService) {
+        final Map<BlackDuckProject, ProjectView> projectMap = new ConcurrentHashMap<>();
+        projectViews
+            .parallelStream()
+            .forEach(projectView -> {
+                String projectOwnerEmail = null;
+                if (StringUtils.isNotBlank(projectView.projectOwner)) {
+                    try {
+                        UserView projectOwner = hubService.getResponse(projectView.projectOwner, UserView.class);
+                        projectOwnerEmail = projectOwner.email;
```

Comment:
The 'projectOwner' field is a link we have to follow to get the user information